worker/src/RTC/Producer.cpp

```diff
@@ -657,10 +663,17 @@ namespace RTC
 			MS_ABORT("found stream does not match received packet");
 		}
 
+
+
+				static int counter = 0;
+		if (packet->GetSsrc() != rtpStream->GetRtxSsrc() && counter++ % 8 == 0)
```

Comment:
discard 1/8th of the packets to simulate packet loss (don't discard rtx packets)